src/cascadia/TerminalApp/AppCommandlineArgs.cpp

```diff
@@ -157,6 +157,7 @@ int AppCommandlineArgs::_handleExit(const CLI::App& command, const CLI::Error& e
 // - <none>
 void AppCommandlineArgs::_buildParser()
 {
+    // -v,--version

```

Comment:
You commented this one with the arguments and you commented the below one with a textual explanation....

Consistency? Maybe do both for both? 

```
//-v,--version: Displays version info
//-M,--maximized: Maximizes the window on launch
//-F,--fullscreen: Fullscreens the window on launch
```